src/main/java/com/parsing/parsers/pdf/parsing/PageOCRPreparator.java

```diff
@@ -118,12 +117,11 @@ private Rectangle getPageMainRectangle(PDDocument document, int pageNumber) {
         ITesseract itesseract = new Tesseract();
         itesseract.setDatapath(getTessDataPath());
         itesseract.setLanguage("ukr+eng");
-        List<Word> result = new ArrayList<>();
-        BufferedImage bim = null;
+        List<Word> result;
         try {
             BufferedImage bufferedImage = pdfRenderer.renderImageWithDPI(pageNumber, 300, ImageType.RGB);
             result = itesseract.getWords(bufferedImage, ITessAPI.TessPageIteratorLevel.RIL_BLOCK);
-            if (result.size() > 0) {
+            if (!result.isEmpty()) {
```

Comment:
get rid of not related to your tasks changes. You need to update your branch from branch where you're doing a PullRequest.

Reply:
it was part of my task